frappe/installer.py

```diff
@@ -80,7 +80,7 @@ def _new_site(
 	)
 
 	for app in apps_to_install:
-		install_app(app, verbose=verbose, set_as_patched=not source_sql)
+		install_app(app, verbose=verbose, set_as_patched=not source_sql, force=force)
```

Comment:
tbh this is not really needed over here :P 
as at that stage we've created a fresh db - but should be fine to leave as it is (i think 😬 ).
(my bad)